src/main/java/Duke.java

```diff
@@ -1,10 +1,26 @@
+import controllers.Ui;
+import models.Storage;
+
 public class Duke {
+    /** Constant to store the database path for Duke's commands */
+    private static final String DATABASE_FILE_PATH = "data/duke.txt";
+    /** Constant storing database directory path */
+    private static final String DATABASE_DIRECTORY_PATH = "data/";
+    /** Storage object to be passed to AppController for reading / writing to db */
+    private final Storage storage;
+
```

Comment:
I like that constant names have been capitalised to follow coding standards and that Javadoc of class members have been specified in a single line. 😄

Reply:
Thanks Clarlzx for your feedback, I will take your feedback into consideration and implement the relevant changes. 👍